pkg/configuration/constants.go

```diff
@@ -13,6 +13,7 @@ const (
 	INTEGRATION_ENVIRONMENT_VERSION string = "snyk_integration_environment_version"
 	ANALYTICS_DISABLED              string = "snyk_disable_analytics"
 	ORGANIZATION                    string = "org"
+	ORGANIZATION_SLUG               string = "org_slug"
```

Comment:
Suggestion: add `internal_` as we are not expecting this to be a explicit externally set value.